Sources/Views/InputTextView.swift

```diff
@@ -239,18 +239,16 @@ open class InputTextView: UITextView {
     
     open override func paste(_ sender: Any?) {
         
-        guard let image = UIPasteboard.general.image else {
+        guard isImagePasteEnabled, let image = UIPasteboard.general.image else {
             return super.paste(sender)
         }
-        if isImagePasteEnabled {
-            pasteImageInTextContainer(with: image)
```

Comment:
`isImagePasteEnabled = true` is needed for this method to get called by the system.